manifest/dev.hazre/ResonitePrivacyShield/info.json

```diff
@@ -0,0 +1,28 @@
+{
+	"name": "ResonitePrivacyShield",
+	"id": "dev.hazre.ResonitePrivacyShield",
+	"description": "Adds Some privacy enhancements like requesting web permission for anything besides Resonite APIs, as well as timezone and FPS spoofing. (Port of PrivacyShield mod by ljoonal)",
+	"category": "Misc",
+	"sourceLocation": "https://github.com/hazre/ResonitePrivacyShield",
+	"website": "http://hazre.dev",
```

Comment:
Yeah I own the domain, It's just I haven't made a page for it, so it just says "parked" for now. It's fine though. I can just remove it, since it's not important.